docs/_includes/date-picker.html

```diff
@@ -0,0 +1,212 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
```

Comment:
Can we add a note saying that arrow key navigation is not implemented in this example. It would actually be really nice to have it, but we don't have 2d capability in makeup-navigation-emitter yet.